src/view/styles/Clue.css

```diff
@@ -129,3 +129,7 @@
 .cls-22 {
     stroke: #7e7272;
 }
+.sim-text-outside {
+    font-size: 14px;
+    fill: var(--vscode-descriptionForeground);

```

Comment:
Can we make this the same color as the "Currently playing" text at the top? For High contrast mode, it is still gray but it would be better if it was white.

Reply:
Yep, will make it foreground